app/main.py

```diff
@@ -1,8 +1,61 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: int,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        if self.clean_power > car.clean_mark:
+            cleaning = (self.clean_power - car.clean_mark)
+            price = (
+                car.comfort_class
+                * cleaning
+                * self.average_rating
+                / self.distance_from_city_center
+            )
+            return round(price, 1)
+        return 0.0
+
+    def wash_single_car(self, car: Car) -> None:
+        if self.clean_power > car.clean_mark:
+            car.clean_mark = self.clean_power
+
+    def rate_service(self, assessment: int) -> None:
+        new_rate = (
+            (
+                self.average_rating
+                * self.count_of_ratings
+                + assessment
+            )
+            / (self.count_of_ratings + 1)
+        )
+        self.average_rating = round(new_rate, 1)
+        self.count_of_ratings += 1
+
+    def serve_cars(self, cars: list) -> float:
```

Comment:
Indicate that you expect to obtain a list of `Car` class instances for `cars` in the type annotation in the future